features/support/env.rb

```diff
@@ -0,0 +1,15 @@
+require 'pry'
+require 'singleton'
+require 'httparty'
+
+require Dir.pwd + '/features/support/apis/generic.rb'
+
+# login credentials should be passed in through the terminal
```

Comment:
suggestion, we could use a file to store these... In test that is fine as they are within our private network. 
For Demo we could use that file but require that it is editted and also in the gitignore.